doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -602,6 +602,15 @@ Networking samples
     The certificate is provisioned only if the :ref:`CONFIG_SAMPLE_SECURE_SOCKET <CONFIG_SAMPLE_SECURE_SOCKET>` Kconfig option is set to ``y``.
   * Fixed an issue where the network interface was not re-initialized after a fault.
 
+* Added support for nRF7002EK with ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples:
```

Comment:
```suggestion
* Added support for nRF7002 EK with ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples:
```
This could be moved to top as the first entry as this affects many samples.